feature-libs/organization/administration/components/unit/form/unit-form.component.html

```diff
@@ -60,6 +60,20 @@
         'orgUnit.approvalProcess' | cxTranslate
       }}</span>
       <ng-select
+        *cxFeature="'a11yApprovalProcessWithNoClearable'"
+        [inputAttrs]="{ required: 'true' }"
+        formControlName="code"
+        [searchable]="false"
+        [clearable]="false"
```

Comment:
Can we just remove this functionality? Maybe this input can be cleared by design?

Reply:
We should keep them.